cvm_deployment/key/skr-policy.json

```diff
@@ -3,119 +3,464 @@
+              "equals": "tdxvm"
+            },
+            {
+              "claim": "x-ms-attestation-type",
+              "equals": "sevsnpvm"
+            }
+          ]
         }
       ],
-      "authority": "https://sharedswn.swn.attest.azure.net/"
+      "authority": "https://sharedcac.cac.attest.azure.net/"
+    },
+    {
+      "allOf": [
+        {
+          "claim": "x-ms-compliance-status",
+          "equals": "azure-compliant-cvm"
+        },
+        {
+          "anyOf": [
+            {
+              "claim": "x-ms-attestation-type",
+              "equals": "tdxvm"
+            },
+            {
+              "claim": "x-ms-attestation-type",
+              "equals": "sevsnpvm"
+            }
+          ]
+        }
+      ],
+      "authority": "https://sharedukw.ukw.attest.azure.net/"
     },
     {
       "allOf": [
         {
-          "claim": "x-ms-attestation-type",
-          "equals": "sevsnpvm"
+          "claim": "x-ms-compliance-status",
+          "equals": "azure-compliant-cvm"
         },
+        {
+          "anyOf": [
+            {
+              "claim": "x-ms-attestation-type",
+              "equals": "tdxvm"
+            },
+            {
+              "claim": "x-ms-attestation-type",
+              "equals": "sevsnpvm"
+            }
+          ]
+        }
+      ],
+      "authority": "https://sharedeau.eau.attest.azure.net/"
+    },
+    {
+      "allOf": [
         {
           "claim": "x-ms-compliance-status",
           "equals": "azure-compliant-cvm"
+        },
+        {
+          "anyOf": [
+            {
+              "claim": "x-ms-attestation-type",
+              "equals": "tdxvm"
+            },
+            {
+              "claim": "x-ms-attestation-type",
+              "equals": "sevsnpvm"
+            }
+          ]
         }
       ],
-      "authority": "https://shareditn.itn.attest.azure.net/"
+      "authority": "https://sharedncus.ncus.attest.azure.net/"
+    },
+    {
+      "allOf": [
+        {
+          "claim": "x-ms-compliance-status",
+          "equals": "azure-compliant-cvm"
+        },
+        {
+          "anyOf": [
+            {
+              "claim": "x-ms-attestation-type",
+              "equals": "tdxvm"
+            },
+            {
+              "claim": "x-ms-attestation-type",
+              "equals": "sevsnpvm"
+            }
+          ]
+        }
+      ],
+      "authority": "https://sharedsau.sau.attest.azure.net/"
+    },
+    {
+      "allOf": [
+        {
+          "claim": "x-ms-compliance-status",
+          "equals": "azure-compliant-cvm"
+        },
+        {
+          "anyOf": [
+            {
+              "claim": "x-ms-attestation-type",
+              "equals": "tdxvm"
+            },
+            {
+              "claim": "x-ms-attestation-type",
+              "equals": "sevsnpvm"
+            }
+          ]
+        }
+      ],
+      "authority": "https://sharedjpw.jpw.attest.azure.net/"
     }
```

Comment:
missing: sharedsan.san.attest.azure.net, shareduaen.uaen.attest.azure.net, sharedwus2.wus2.perf.attest.azure.net

Reply:
MAA team confirmed that these 2 endpoints not to be included.